robot_link.py

```diff
@@ -18,6 +18,15 @@ def __init__(self, robot):
         self.joint_maxs = []
         self.methods_bound = [0, 0, 0, 0]
 
+        #Support for Collision Checking
+        self.link_names = None
+        self.vis_props = None #Used as a backup if col props unavailable
+        self.col_props = None #Populate first
+        #Col/Vis Props Format:
```

Comment:
These values are not unused, they are filled in by the URDF, but because there are other means to generate arms and the URDF may not have values for each, it is possible to leave them None and have the code still function. col_rops is necessary for collisions for instance.

Reply:
sorry, I'm not sure what I was referencing.